src/components/inputs/rich-text-input/plugins/list.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
I think the inner working of the RichText (as a follow up) could use some architecture readme of how things interact so it's a bit easier for somebody else to add something if needed.

Reply:
yeah, that would be a good followup